autonav_ws/src/autonav_hardware/src/audible_feedback.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+
+import rclpy
+from autonav_msgs.msg import AudibleFeedback
+
+from autonav_shared.node import Node
+from autonav_shared.types import LogLevel, DeviceState, SystemState
+import time
+import os
+from just_playback import Playback
+
+# TODO: rewrite with just_playback
+
+class AudibleFeedbackConfig:
+    def __init__(self):
+        self.volume = 100
+        
+
+class AudibleFeedbackNode(Node):
+    def __init__(self):
+        super().__init__("audible_feedback_node")
+        self.write_config(AudibleFeedbackConfig())
+        self.current_playing_thread = None
+        self.tracks = []
+        self.old_system_state = self.system_state
+
+        # 
+        self.audible_feedback_subscriber = self.create_subscription(
+            AudibleFeedback, 
+            "/autonav/audible_feedback",
+            self.on_audible_feedback_received,
+            20
+        )
+
+        self.system_state_monitor = self.create_timer(0.5, self.monitor_system_state)
+        self.track_monitor = self.create_timer(0.5, self.monitor_tracks)
+
+    def on_audible_feedback_received(self, msg:AudibleFeedback):
+        self.monitor_tracks()
+
+        self.log(f"{len(self.tracks)}", LogLevel.ERROR)
+        if msg.stop_all:
+            self.stop_all()
+
+        else:
+            # self.log(f"playing {msg.filename}")
+            filename = str(msg.filename)
+
+            # self.log(f"heard request to play {msg.filename}")
+            self.play_sound(filename)
+
+
+    def play_sound(self, filename):
+        playback = Playback()
+        playback.load_file(filename)
```

Comment:
if this file doesn't exist does this fail gracefully or crash the code? if it crashes a try/catch would be nice, otherwise no problems, although doing some processing on the file name (like a .strip() maybe) might be worth it?